Sources/SKSupport/dlopen.swift

```diff
@@ -10,7 +10,14 @@
 //
 //===----------------------------------------------------------------------===//
 
-import TSCLibc
```

Comment:
There's a _reference_ to TSCLibc on line 92 that will need to be changed
```swift
 guard let handle = TSCLibc.dlopen(path, mode.rawValue) else {
```

Reply:
Ah, thanks; that actually is the re-exported libc function, so we should be able to just drop `TSCLibc.`.